doc/rados/operations/cache-tiering.rst

```diff
@@ -1,6 +1,9 @@
 ===============
  Cache Tiering
 ===============
+.. warning:: Cache tiering has been deprecated in the Reef release as it
+             lacks a maintainer for a very long time. This does not mean
```

Comment:
s/lacks/has lacked/
below:  s/for that/to do so/

Reply:
Fixed.